satpy/composites/__init__.py

```diff
@@ -1014,6 +1014,46 @@ def zero_missing_data(data1, data2):
     return data1.where(~nans, 0)
 
 
+class PrecipCloudsRGB(GenericCompositor):
+
+    def __call__(self, projectables, *args, **kwargs):
+        """Make an RGB image out of the three probability categories of the NWCSAF precip product"""
```

Comment:
```suggestion
        """Make an RGB image out of the three probability categories of the NWCSAF precip product."""
```

Don't forget the `.`

Reply:
Ok, done!